libs/system-admin-pages/src/assets/css/dashboard.scss

```
@@ -40,8 +40,33 @@
+
+.audit-log-layout__content.govuk-grid-column-three-quarters {
+  padding-right: 0;
+}
+
+.audit-log-layout__content {
+  flex: 1;
+  min-width: 0;
+  overflow-x: auto;
+
+  .audit-log-layout__word-break {
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)